### ISSUE TITLE
NR==FNR check fails if I use code from a file using -f option

### ISSUE THREAD

**Issue by learnbyexample** [ezrosent/frawk#59]
I'm not sure if this is already covered in https://github.com/ezrosent/frawk/issues/24

This is a simplified failing case:

```bash
$ seq 3 > f1
$ seq 10 15 > f2

# works as expected
$ frawk 'NR==FNR{print NR, FNR}' f1 f2
1 1
2 2
3 3

# write the code in a file
$ echo 'NR==FNR{print NR, FNR}' > test.frawk
$ cat test.frawk
NR==FNR{print NR, FNR}
# code fails
$ frawk -f test.frawk f1 f2
1 1
2 2
3 3
4 4
5 5
6 6
```

In case it makes a difference, this is how I installed:

```bash
sudo apt install rustc # version 1.47.0
cargo install frawk --no-default-features --features use_jemalloc,allow_avx2
```

**Comment by ezrosent** [comment]
Hey, thanks for catching this issue, and for taking the time to provide a simple test case. 

I believe I have a fix. I should have it pushed soon.

**Comment by ezrosent** [comment]
I believe this is resolved at master; and in frawk 0.4.1 on crates.io.
